packages/components/src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import css from '@styled-system/css';
+import { User } from '@codesandbox/common/lib/types';
+import styled from 'styled-components';
+import { Element } from '../Element';
+import { Text } from '../Text';
+
+interface IAvatarProps {
+  user: User;
+}
+
+export const AvatarImage = styled.img(
+  css({
+    width: 8,
+    height: 8,
+    borderRadius: 'small',
+    border: '1px solid',
+    borderColor: 'sideBar.border',
+    backgroundColor: 'sideBar.border', // fallback for loading image
+  })
+);
+
+export const Pro = styled(Text).attrs({ size: 1, weight: 'bold' })(
+  css({
+    backgroundColor: 'blues.700',
+    color: 'white',
+    borderRadius: 'small',
+    paddingX: 2,
+    paddingY: '2px', // no 2 in spaces
+    border: '1px solid',
+    borderColor: 'sideBar.border',
```

Comment:
(after previous comment ^) from what I understood from danny, the badge border should be same as the sidebar background

Reply:
I asked on slack, its gray 500

He updated the design :)